server/apis/v1/handler.go

```diff
@@ -322,6 +322,25 @@ func (h *handler) GetPipelineStatus(c *gin.Context) {
 	c.JSON(http.StatusOK, l)
 }
 
+// ListNamespaces is used to provide all the namespaces that have numaflow pipelines running
+func (h *handler) ListNamespaces(c *gin.Context) {
+	ns := c.GetString("namespace")
+	l, err := h.numaflowClient.Pipelines(ns).List(context.Background(), metav1.ListOptions{})
```

Comment:
The `ns` here should be always empty string.

Reply:
right, it was the previous implementation i didn't modify it